src/applications/personalization/search-representative/config/search/searchRepresentativeResult.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const searchRepresentativeResult = props => {
+  const customStyle = {
+    maxWidth: 180,
+  };
+  const linkStyles = {
+    borderRadius: 0,
+    textDecoration: 'underline',
+  };
+  return (
+    <div className="vads-u-margin-bottom--1p5 vads-u-background-color--gray-lightest vads-u-padding--1p5">
+      <p className="vads-u-font-size--h3 vads-u-font-family--serif vads-u-font-weight--bold">
+        {props.option.name}
+      </p>
+      <p className="vads-u-font-weight--bold">{props.option.type}</p>
+      <p>{props.option.address}</p>
+      <p>{props.option.city}</p>
+      <p>{props.option.phone}</p>
+      <div style={customStyle}>
+        <button
+          style={linkStyles}
```

Comment:
You should be able to remove this. The alterations made are already covered by the `va-button-link` class

Reply:
Yep, removed